reconcile/test/fixtures/templating/full.yaml

```diff
@@ -1,7 +1,9 @@
 template:
   name: full
 
-  targetPath: /{{bar}}/foo.yml
+  # test targetPath with templating, multiline and whitespaces
+  targetPath: |
+    {% if true %} /{{bar}}/foo.yml {% endif %}
```

Comment:
maybe
```
{% if true %}
      /{{bar}}/foo.yml     # with ending spaces
{% endif %}
```

Reply:
applied in [6916c35](https://github.com/app-sre/qontract-reconcile/pull/4499/commits/6916c354fdd911569dc41eb58d35604a8074905d)